script/Arcueid/Day 11/11_00_ARC11_2.txt

```diff
@@ -1513,7 +1528,7 @@ The thing she shortened her own lifespan to protect.
 -- Page 135, Offset 29889.
 -- 本来、何よりも優先するべき、自分たちの幸福の在処。
 The thing we naturally ought to prioritize above all else:
- the whereabouts of our own happiness.
+ Knowing where we can find our own happiness.
```

Comment:
You know, this came solely down to the new line for me... but this 'usage case' is extraordinarily rare in modern English.
At the same time I just realized that this may not be a true line break? Maybe I'm misreading the way the engine handles the text.

At the same time, I think this strict "capitalization on new line after punctuation (whether that's a colon or a dot)" is like kind of archaic anyway, so maybe I've just been reading too much Milton.

Reply:
Newlines in the script text are ignored, they only serve to make the text easier to read when you're working in it. To actually force a line break in game you need an `%{n}`, otherwise the game will just break the lines so that they wrap properly (one row on the screen is 55 chars max).